_posts/2019-08-20-Introducing-bundler-leak.markdown

```diff
@@ -0,0 +1,42 @@
+---
+layout: post
+title: Introducing bundler leak
+date: 2019-08-20 12:53:28
+categories: ["bundler"]
+author: bronzdoc
+
+---
+In this blog post I will introduce bundler leak - A bundler plugin to check gems affected by memory leaks in your Gemfile.
+
+<!--more-->
+
+The [bundler leak](https://github.com/rubymem/bundler-leak) plugin is a fork of the famous [bundler audit](https://github.com/rubysec/bundler-audit) plugin.
+
+As bundler audit, bundler leak works in conjuction of a couple of projects. The first one is what we called the [ruby-mem-advisory-db](https://github.com/rubymem/ruby-mem-advisory-db) a text based database
+consisting of gems with known memory leak issues. Bundler leak will compare gems stored in this database against your Gemfile i.e
```

Comment:
@bronzdoc Please make sure that you mention that this all started thanks to https://github.com/ASoftCo/leaky-gems